lib/v2/components/scanner/interactor/viewmodels/scanner_state.dart

```diff
@@ -1,22 +1,22 @@
 import 'package:equatable/equatable.dart';
 
-enum PageState { scan, processing, success, stop }
+enum ScanStatus { scan, processing, success, stop }
```

Comment:
I changed the name to a more meaningful one in addition to avoiding importing this enum sometimes instead of the global PageState.

Reply:
gracias